docs/common-problems.asciidoc

```diff
@@ -212,3 +213,25 @@ the APM server and Elasticsearch log a warning like this:
 {\"type\":\"illegal_argument_exception\",\"reason\":\"Limit of total fields [1000] in index [apm-7.0.0-transaction-2017.05.30] has been exceeded\"}
 ----
 
+[[io-timeout]]
+[float]
+=== I/O Timeout
+
+I/O Timeouts can occur when your timeout settings across the stack are not configured correctly,
+especially when using a load balancer.
+The following error may appear in the client command line:
+
+----------------------------------------------------------------------
+[ElasticAPM] APM Server responded with an error:
+"read tcp 123.34.22.313:8200->123.34.22.40:41602: i/o timeout"
```

Comment:
this is from the APM agents perspective. It will show up in the logs. A similar error will show up on the APM Server side.